app/components/summary_card_component.rb

```diff
@@ -13,4 +13,15 @@ def border_css_class
 private
 
   attr_reader :rows
+
+  def determine_rows(rows, editable)
+    rows.map do |row|
+      row.tap do |r|
```

Comment:
What's the `tap` doing here?

Reply:
`.tap` pretty cool. In this case, it returns the object `r` at the end so we don't have to do an explicit return of it.

See https://apidock.com/ruby/Object/tap.